### DISCUSSION TITLE
Capstone - tidak bisa mengubah tipe data

### DISCUSSION THREAD

**Discussion by moogli90** [teamalgoritma/community#203] [Q&A] [answered]
bagaimana cara agar dapat mengubah tipe data kolom "vote" menjadi int agar bisa dilakukan perhitungan?
![image](https://user-images.githubusercontent.com/107478291/178374781-8e2468ac-7d13-4fe3-94c5-7ddad0519a8b.png)

**Comment by triarm** [comment, marked as answer]
Hi @moogli90 ,
Error tersebut terjadi karena tanda koma (,) pada kolom vote belum terhapus. Untuk menghapus tanda tersebut, Anda dapat mencoba kode berikut:
```
data['vote'] = data['vote'].str.replace(",","")
```

Semoga membantu

**Reply by moogli90** [reply]
bagaimana membuat nama label menjadi miring?
![image](https://user-images.githubusercontent.com/107478291/178398899-c0f57d5f-8e9a-4f22-a0f8-36f464ac2ac7.png)

**Reply by moogli90** [reply]
trimkasih bantuanya kak, 
ijin mo nanya lg
ini kenapa tidak bisa ya?
![image](https://user-images.githubusercontent.com/107478291/178418522-df130264-70ba-41ae-8a13-062755d0b8e5.png)

**Reply by triarm** [reply]
Hi @moogli90 ,
terkait pertanyaan tersebut

1. Untuk merotasi label pada sumbu x, Anda dapat menambahkan parameter `rot` dengan sudut yang diinginkan. Misal
```
populer2.plot(rot = 45)  #membuat label sumbu x miring 45 derajat
```

2. Terkait error yang kedua, Apakah Anda sudah memastikan package-package yang ada pada file requirements.txt sudah terinstall dengan baik pada environment tersebut? Anda dapat memeriksanya dengan perintah `pip list` kemudian periksa satu per satu package apakah package dari requirements sudah ada atau belum

Demikian yang dapat kami sampaikan. Semoga menjawab

**Reply by moogli90** [reply]
terimakasih bantuannya kak, capstone sudah sy kirim
